dev-docs/Development.md

```diff
@@ -5,13 +5,29 @@
 For Golang, we try to follow [Google's code style](https://google.github.io/styleguide/go/)
 For Python we try to follow [PEP8](https://peps.python.org/pep-0008/) style guid
 
+### Hermit
+
+We use [hermit](https://cashapp.github.io/hermit/) to manage development environment.
```

Comment:
```suggestion
We use [hermit](https://cashapp.github.io/hermit/) to manage the development environment.
```

Reply:
I'll collect everything under one file! I did not see the README.hermit.md. It actually covers it all. I'll just point to it